### PR TITLE
[#176415651] renamed package to @pagopa/ts-commons

### PR DESCRIPTION
**Pull request by gquadrati**
renamed package from _"italia-ts-commons"_ to _"@pagopa/ts-commons"_

### CODE REVIEW

**Comment by gquadrati** [comment]
NOTE: I was not able to re-generate the docs

**Comment by balanza** [comment]
@gquadrati Should we take the opportunity to review the readme?

**Comment by gquadrati** [comment]
> @gquadrati Should we take the opportunity to review the readme?

Yes, I wasn't able to understand how to generate the docs, since "yarn docs" didn't do the job..

 But first of all... I was checking right now that the Azure pipeline didn't started :/

**Comment by balanza** [comment]
idk if we actually use such docs ;)

**Comment by gquadrati** [comment]
> But first of all... I was checking right now that the Azure pipeline didn't started :/

Maybe because there's no _.yml_ file in this branch, Greta.. -.-"

File: README.md [hunk 1]
```diff
@@ -1,37 +1,93 @@
...
 ```
 $ yarn docs
 ```
 
-Then git add/commit/push.
 
 ### Releasing a version
 
-Make sure you update the docs first.
+Release has been automatized by using Azure Pipelines.
```
Comment (balanza) [inline README.md:87]: We should think about make doc generation automated by a pipeline. Anyway, that's very low priority.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,93 @@
...
+[![Build Status](https://dev.azure.com/pagopa-io/io-ts-commons/_apis/build/status/pagopa.io-ts-commons.code-review?branchName=refs%2Fpull%2F343%2Fmerge)](https://dev.azure.com/pagopa-io/io-ts-commons/_build/latest?definitionId=74&branchName=refs%2Fpull%2F343%2Fmerge)
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fteamdigitale%2Fitalia-ts-commons.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fteamdigitale%2Fitalia-ts-commons?ref=badge_shield)
 
 [![codecov](https://codecov.io/gh/teamdigitale/italia-ts-commons/branch/master/graph/badge.svg)](https://codecov.io/gh/teamdigitale/italia-ts-commons)
 
 [![Maintainability](https://api.codeclimate.com/v1/badges/c9be630a66618bde8e4a/maintainability)](https://codeclimate.com/github/teamdigitale/italia-ts-commons/maintainability)
 
-# Italia TypeScript Commons
+# PagoPA TypeScript Commons
```
Comment (balanza) [inline README.md:9]: I think just TypeScript Commons does the job
```suggestion
# TypeScript Commons
```

**Review by balanza — APPROVED** [review]
Great job! Lgtm besides minor comments. When you feel this PR is ready, we can merge.

File: README.md (hunk 1 above)
Comment (gquadrati) [reply README.md:87]: Agree.. like a scheduled pipeline that generates and merges doc by itself?

Comment (balanza) [reply README.md:87]: We could do that. But again: is very low priority.

**Review by balanza — APPROVED** [review]

**Pull request merged**